src/liboslexec/opnoise.cpp

```diff
@@ -704,6 +704,13 @@ struct GenericNoise {
                    || name == STRING_PARAMS(snoise)) {
             SNoise snoise;
             snoise(result, s);
+        } else if (name == STRING_PARAMS(cell)) {
+            CellNoise cellnoise;
+            cellnoise(result.val(), s.val());
+            result.clear_d();
+        } else if (name == STRING_PARAMS(gabor)) {
+            GaborNoise gnoise;
+            gnoise(name, result, s, sg, opt);
```

Comment:
What's going on here?

Reply:
This is a workaround for the OptiX parse error with invalid global names. Changing the order of if statements like this somehow avoids it.

Hopefully it will not be needed once there is a proper fix.

Reply:
That's alarmingly spooky.